README.md

```diff
@@ -95,12 +95,13 @@ An official guide by Nvidia can be found [here](https://docs.nvidia.com/datacent
 
 ### Version Table
 
-| Docker tag                | Blender  | CUDA |
-|:-------------------------:|:--------:|:----:|
-| `nvidia-beta`             | 2.93 LTS | 10.2 |
-| `bl_2.83_cu_10`, `nvidia` | 2.83 LTS | 10.2 |
-| `bl_2.79_cu_10.1`         | 2.79b    | 10.1 |
-| `bl_2.79_cu_9.2`          | 2.79b    | 9.2  |
+| Docker tag                   | Blender  | CUDA |
+|:----------------------------:|:--------:|:----:|
+| `nvidia-beta`                | 2.93 LTS | 10.2 |
+| `bl_2.83_cu_11`, `nvidia_11` | 2.83 LTS | 11.0 |
```

Comment:
```suggestion
| `bl_2.83_cu_11`, `nvidia`    | 2.83 LTS | 11.0 |
```

Reply:
Hi TrojanCoder, thanks for this PR 💯   😄 

Agree with Zocker that we should reduce the number of tags, this might help users by reducing the number of options they need to consider. Otherwise, looks good to me, and thanks again for contributing !!